### PR TITLE
Simplify attributes

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)

### CODE REVIEW

File: guides/common/attributes-orcharhino.adoc [hunk 1]
```diff
@@ -4,22 +4,22 @@
 :ProductVersion: 6.2
 :ProductVersionPrevious: 6.1
 :Project: orcharhino
-:ProjectName: orcharhino
-:ProjectNameX: orcharhino
-:ProjectNameXY: orcharhino {ProductVersion}
-:ProjectServer: orcharhino Server
+:ProjectName: {Project}
+:ProjectNameX: {Project}
+:ProjectNameXY: {Project}{nbsp}{ProductVersion}
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/attributes-orcharhino.adoc]: Removed.

File: guides/common/attributes-orcharhino.adoc [hunk 2]
```diff
@@ -4,22 +4,22 @@
...
 :ProjectServerTitle: {Project}{nbsp}Server
 :ProjectVersion: {ProductVersion}
 :ProjectVersionPrevious: {ProductVersionPrevious}
 :ProjectWebUI: {Project} management UI
-:ProjectX: orcharhino
-:ProjectXY: orcharhino {ProductVersion}
-:Project_Link: orcharhino
+:ProjectX: {Project}
+:ProjectXY: {Project}{nbsp}{ProductVersion}
+:Project_Link: {Project}
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/attributes-orcharhino.adoc]: Removed.

File: guides/common/attributes-orcharhino.adoc [hunk 3]
```diff
@@ -4,22 +4,22 @@
...
 :RHEL: Red Hat Enterprise Linux
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/attributes-orcharhino.adoc]: Makes sense; removed.

File: guides/upstreamize.rb
Comment (ekohl) [inline guides/upstreamize.rb:1]: I think all guides except Hammer are upstreamed. I'm looking at @adamlazik1 and @Lennonka now, but should we drop this file?

Comment (adamlazik1) [reply guides/upstreamize.rb:1]: Actually, we have the following guides that are not upstreamized:

* [API guide](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html-single/api_guide/index)
* [Configuring Virtual Machine Subscriptions in Red Hat Satellite](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html-single/configuring_virtual_machine_subscriptions_in_red_hat_satellite/index)
* [Hammer Cheat Sheet](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html-single/hammer_cheat_sheet/index)
* [Hammer CLI Guide](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html-single/hammer_cli_guide/index)
* [Package Manifest](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html-single/package_manifest/index)
* [Release Notes](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html-single/release_notes/index) - Not guide but I wonder if it should be upstreamized too or not?

That being said, I have not seen an initiative to upstreamize any of these in the past six months at least. There was a brief mention of upstreamizing hammer guide but we seemed to forgot about it. Maybe I can bring it up on internal Satellite writer call to see if we want to upstreamize any of these?

**Review by adamlazik1 — APPROVED** [review]
Changes look good but take extreme care when cherry-picking due to removed attributes.

File: guides/upstreamize.rb
Comment (ekohl) [reply guides/upstreamize.rb:1]: Please do. If we don't want to upstream them, then we can drop the script.

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   38ddc1bd4..bf38786a0  3.6 -> 3.6
   8dc215e0e..82fdd3fc3  3.5 -> 3.5
   866e5fd98..c972a4650  3.4 -> 3.4
   62c47f363..6e2c3e53b  3.3 -> 3.3

3.5, 3.4, and 3.3 had minimal conficts in "attributes-orcharhino.adoc" which I've resolved locally.

**Pull request merged**